substratevm/src/com.oracle.svm.test.jdk11/src/com/oracle/svm/test/jdk11/jfr/utils/poolparsers/PackageConstantPoolParser.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, 2021, Red Hat Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.oracle.svm.test.jdk11.jfr.utils.poolparsers;
+
+import com.oracle.svm.test.jdk11.jfr.utils.RecordingInput;
+
+import java.io.IOException;
+
+public class PackageConstantPoolParser implements ConstantPoolParser {
+
+    @Override
+    public void parse(RecordingInput input) throws IOException {
+        long numberOfPackages = input.readLong();
+        for (int i = 0; i < numberOfPackages; i++) {
+            // TODO: Asserts.
```

Comment:
Please try to get rid of all TODOs. At least basic assertion that check the value range (e.g., value > 0) should always be possible. More checks can always be added later on.

Reply:
All TODOs are removed.